### DISCUSSION TITLE
Tools and CI meeting

### DISCUSSION THREAD

**Discussion by Ashwin-Veeraiah** [tianocore/edk2#5787] [Q&A] [answered]
Greetings!
How do you add an item to be discussed in the Tools and CI meeting. 
Currently the meeting is set to occur every Monday, when is the post announcing the meeting posted?
Do we reply to that post? Or do we reply in the edk2.io groups calendar invite?
Thanks!

**Comment by makubacki** [comment, marked as answer]
Hi @Ashwin-Veeraiah, @spbrogan chairs the Tools and CI Meeting. The last published guidance is in [this post](https://github.com/tianocore/edk2/discussions/2614#discussioncomment-7782629).

Today I will be covering for @spbrogan and the meeting will be held. Per those instructions, I've created the thread for today in [Tools and CI Meeting - Date 6/24](https://github.com/tianocore/edk2/discussions/5818). Please respond to that thread with your topic. There should also be enough time to cover topics raised in the meeting as well.

**Reply by makubacki** [reply]
@Ashwin-Veeraiah, that's the process at this time, though it has not been followed consistently. We can propose a change to the process if desired, but for now I'm going to mark this specific discussion resolved.
